main.py

```diff
@@ -24,62 +26,78 @@
 
 
 class Item:
-    def __init__(self, title, price, torg, url, date, image):
+    def __init__(self, title, price, torg, url, image):
         self.title = title
         self.price = price
         self.torg = torg
         self.url = 'https://www.ebay-kleinanzeigen.de' + url
-        self.date = date
+        #self.date = date
```

Comment:
Why do you remove date?

Reply:
I didn't find a place within the article where the date can be grabbed. The format is (often) something like "Today 12:05". We might need more logic to support that

Reply:
Ok, I still think date is needed, we can easily parse heute/gestern, and for other blocks it's regular date, so it should work